README.md

```diff
@@ -1,16 +1,36 @@
 
 
+
 ![](https://img.shields.io/badge/Microverse-blueviolet)
+![screenshot](screenshot.png)
 
 # Tic Tac Toe
 
-  On this milestone we created the lib and bin folder, as well as creating the main.rb file.
+  On this project we created a copy of the tic tac toe game, to be player by 2 players using Ruby, the game follows the standard rules of tic tac toe which are explained in the rules section.
```

Comment:
**[MANDATORY]**
- [ ] Add details more to make it more descriptive

**[OPTIONAL]**
- [ ] Pass what you wrote to a grammar checker like Grammarly.